livre/Eq/FrameData.h

```diff
@@ -30,12 +30,15 @@ namespace livre
 
 enum SerializationMapIds
 {
-    SMI_FRAME_SETTINGS = 1u,
-    SMI_RENDER_SETTINGS,
-    SMI_CAMERA_SETTINGS,
-    SMI_VOLUME_SETTINGS,
-    SMI_VR_PARAMETERS,
-    SMI_EFP_PARAMETERS
+    SMI_FRAME_SETTINGS = 1u
+    , SMI_RENDER_SETTINGS
+    , SMI_CAMERA_SETTINGS
+    , SMI_VOLUME_SETTINGS
+    , SMI_VR_PARAMETERS
+    , SMI_EFP_PARAMETERS
+#ifdef LIVRE_USE_RESTCONNECTOR
+    , SMI_NETWORK_PARAMETERS
+#endif
```

Comment:
Enum ',' stay where they where before. It's only for initializer lists that they are in front.